tuxemon/core/event/actions/create_npc.py

```diff
@@ -72,7 +73,7 @@ def start(self):
                 slug
             )
         else:
-            sprite = db.JSONDatabase('npc').database['npc'][slug].get('sprite_name')
```

Comment:
sprite = databases.npc.database['npc'][slug].get('sprite_name')

Reply:
Really strange that I wasn't seeing it; I've been able to enter the maple house and talk to the professor's wife just fine. I tested your change and it works great so I committed it: Please double-check that this made the issue go away.

Reply:
Different code path, only affects some npcs